lesson_15/edmond_sabou/test_home1_les13.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+
+class TestPoligons(unittest.TestCase):
+
+    def test_if_side_is_not_none(self):
+        sides = 1, "asd", 3
+
+        self.assertIsNotNone(sides, "Value cannot be of type None")
+
+    def test_if_value_is_not_string(self):
+        x = 1
+        y = 2
```

Comment:
same as the above comment. This test is not tucking the code, the test will never fail

Reply:
I had a feeling something was not ok. changed it now. please check